server/models/prospects.py

```diff
@@ -13,7 +13,8 @@ class ProspectModel(db.Model):
     firstName = db.Column(db.String(120), nullable=True)
     lastName = db.Column(db.String(120), nullable=True)
     userId = db.Column(db.BigInteger, db.ForeignKey("users.id"), nullable=False)
-    # prospect_step_join = db.relationship('ProspectStepModel', backref='prospect_step', lazy=True)
+    prospect_step_join = db.relationship('ProspectStepModel', backref='prospect_step', lazy=True)
+    prospect_step_join = db.relationship('ProspectCampaignModel', backref='prospect_campaign', lazy=True)
```

Comment:
Think maybe u meant to do prospect_campaign_join?

Reply:
Indeed - fixed